pkg/apis/garden/v1beta1/types.go

```diff
@@ -1250,8 +1250,39 @@ type CloudControllerManagerConfig struct {
 // KubeControllerManagerConfig contains configuration settings for the kube-controller-manager.
 type KubeControllerManagerConfig struct {
 	KubernetesConfig `json:",inline"`
+	// HorizontalPodAutoscalerConfig contains horizontal pod autoscaler configuration settings for the kube-controller-manager.
+	// +optional
+	HorizontalPodAutoscalerConfig *HorizontalPodAutoscalerConfig `json:"horizontalPodAutoscaler,omitempty"`
 }
 
+// HorizontalPodAutoscalerConfig contains horizontal pod autoscaler configuration settings for the kube-controller-manager.
+// Note: Descriptions were taken from the Kubernetes documentation.
+type HorizontalPodAutoscalerConfig struct {
+	// The period since last downscale, before another downscale can be performed in horizontal pod autoscaler.
+	// +optional
+	DownscaleDelay *time.Duration `json:"downscaleDelay,omitempty"`
+	// The period for syncing the number of pods in horizontal pod autoscaler.
+	// +optional
+	SyncPeriod *time.Duration `json:"syncPeriod,omitempty"`
+	// The minimum change (from 1.0) in the desired-to-actual metrics ratio for the horizontal pod autoscaler to consider scaling.
+	// +optional
+	Tolerance *float64 `json:"tolerance,omitempty"`
+	// The period since last upscale, before another upscale can be performed in horizontal pod autoscaler.
+	// +optional
+	UpscaleDelay *time.Duration `json:"upscaleDelay,omitempty"`
+}
+
+const (
```

Comment:
must be adapted to to change from `*time.Duration` to *metav1.Duration*

Reply:
Done.

Is there any guidelines about how to update the release notes to highlight that the defaulting for seeds have now been removed and should be done explicitly in the respective seeds' `Shoot` spec?